<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicSettingsFrame</class>
 <widget class="QFrame" name="BasicSettingsFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tolerance</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="toleranceLineEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X Acceleration and Deceleration</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="xADEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y Acceleration and Deceleration</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="yADEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Z Acceleration and Deceleration</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zADEdit"/>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>X Max Speed</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="xMaxSpeedEdit"/>
   </item>
   <item row="1" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Y Max Speed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="yMaxSpeedEdit"/>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Z Max Speed</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="ICLineEditWithVirtualNumericKeypad" name="zMaxSpeedEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ICLineEditWithVirtualNumericKeypad</class>
   <extends>QLineEdit</extends>
   <header>iclineeditwithvirtualnumerickeypad.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
